### DISCUSSION TITLE
Has anybody actually tried sending commands to Deye inverter?

### DISCUSSION THREAD

**Discussion by aleksei-glitch** [StephanJoubert/home_assistant_solarman#502] [Q&A] [answered]
I see people asking. I found the advanced mode but still hesitant on what send first. Any experience? Anything harmless?

**Comment by aleksei-glitch** [comment]
Fingers were itchy and tried without success on my Deye GB-SL15K-EU writing to registers:
154 - Sell mode time point 1 
108 - Max A Charge

The result is that the service "Solarman: Write Holding Register (Modbus Function Code 6)" said everything was OK but no actual changes in registers were made :(

**Comment by K3iVi** [comment, marked as answer]
You have to use the second service Solarman: Write Multiple Holding Registers (Modbus Function Code 16). It contains first zero to enable write or something which I do not understand :D But it works:
![image](https://github.com/StephanJoubert/home_assistant_solarman/assets/80969933/7a63c822-9cc1-4fe1-8883-8e96c3617f49)
 Register 108 changes charging current to 88 on the picture.
